### ISSUE TITLE
Unused variable

### ISSUE THREAD

**Issue by Calculamatrise** [mdn/content#35921]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/API/TextMetrics/fontBoundingBoxAscent

### What specific section or headline is this issue about?

An unused variable in the code example

### What information was incorrect, unhelpful, or incomplete?

A variable, text, was created, but never used. Instead, a new string was created and used to serve its purpose.

### What did you expect to see?

```js
const canvas = document.createElement("canvas");
const ctx = canvas.getContext("2d");

ctx.font = "25px serif";
const text = "Foo";

const textMetrics = ctx.measureText(text); // returns TextMetrics object
const ascentCssPixels = textMetrics.fontBoundingBoxAscent;
```

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/api/textmetrics/fontboundingboxascent`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/TextMetrics/fontBoundingBoxAscent
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/api/textmetrics/fontboundingboxascent/index.md
* Last commit: https://github.com/mdn/content/commit/daeff1a2efaae366bcc8b6d911d86985646e665e
* Document last modified: 2024-04-22T04:54:38.000Z

</details>

**Comment by Josh-Cena** [comment]
If you click "play" you will see the hidden code. That is not displayed here because our usage of this variable is trivial: we just display it as text. The gist of this example is only to show how to retrieve this property, and then it's up to you to decide how to use it.

**Comment by Josh-Cena** [comment]
Wait, you are talking about the variable `text`. It's not unused though; it's used as the subject of measurement.

**Comment by Calculamatrise** [comment]
> Wait, you are talking about the variable `text`. It's not unused though; it's used as the subject of measurement.
@Josh-Cena 


Yes. The variable `text` is defined but never referenced. On the website it appears like this:
```js
const canvas = document.createElement("canvas");
const ctx = canvas.getContext("2d");

ctx.font = "25px serif";
const text = "Foo";

const textMetrics = ctx.measureText("foo"); // returns TextMetrics object
const ascentCssPixels = textMetrics.fontBoundingBoxAscent;
```

**Comment by Josh-Cena** [comment]
I see what you mean. Yes, accepting a PR to use `text`.
